packages/frontend/src/features/catalog/components/CatalogAnalyticCharts.tsx

```diff
@@ -32,28 +38,76 @@ export const CatalogAnalyticCharts: FC<React.PropsWithChildren<Props>> = ({
     }
 
     return (
-        <Stack>
+        <Stack w="100%">
+            <Paper
+                maw="50%"
```

Comment:
Is something else going in the other 50%? It looks a little funny at the moment

Reply:
Great point @magnew, I've actually removed this section as in the intended _final_ design we want 3 cards here: 
Views + Charts + (something else I cant remember)

Since we don't have all 3, then we remove this one.